docs/logs/extending-the-sdk/Program.cs

```diff
@@ -18,6 +18,8 @@
 using Microsoft.Extensions.Logging;
 using OpenTelemetry;
 
+namespace ExtendingTheSdk;
```

Comment:
if we are adding ns, would `OpenTelemetry.Docs.Logs.ExtendingTheSDK` look better?

Reply:
Tagging @reyang as he had some concerns about ns. (I am not sure actually...some vague memory about an old comment about namespaces in docs..)

Reply:
Maybe, but I sort of guessed that the point of not putting these in namespaces in the first place was to minimize boilerplate, so I erred on the side of "keep it super short" rather than having a larger fully-qualified namespace. Happy to change these if folks think that'd be better.

Reply:
This seems okay. Will resolve this.

If there are additional comments, we can address later.

Reply:
I don't have strong opinion, either way looks fine to me.